catalog/ui/search-ui/search-endpoint/src/test/java/org/codice/ddf/ui/searchui/query/controller/SearchControllerTest.java

```diff
@@ -140,7 +139,7 @@
 
     @Before
     public void setUp() throws Exception {
-        framework = createFramework();
+        framework = createFramework(2);
```

Comment:
@mweser IntelliJ will now tell you what an ambiguous parameter name is -- so not knowing what the 2 is there for is just a shortcoming of github reviews 😄